contracts/ronin/validator/storage-fragments/TimingStorage.sol

```diff
@@ -42,7 +42,8 @@ abstract contract TimingStorage is ITimingInfo, GlobalConfigConsumer {
    * @inheritdoc ITimingInfo
    */
   function tryGetPeriodOfEpoch(uint256 _epoch) external view returns (bool _filled, uint256 _periodNumber) {
-    return (_epoch <= epochOf(block.number) || _periodOf[_epoch] > 0, _periodOf[_epoch]);
+    uint256 periodOf = _periodOf[_epoch];
+    return (_epoch <= epochOf(block.number) || periodOf > 0, periodOf);
```

Comment:
Good opt. However, rename the temp var for better meaning.

Reply:
fixed.